lib/rules.json

```diff
@@ -1580,7 +1589,7 @@
                     "dateState": "The document's status and date <span class=\"rfc2119\">must</span> be in a <code>&lt;p id=\"w3c-state\"&gt;</code> element as follows (see also <a href=\"#datesFormat\">date syntax</a>): <pre xml:space=\"preserve\">&lt;p id=\"w3c-state\"&gt;<a href=\"https://www.w3.org/standards/types#@@Profile\">W3C @{param1}</a> DD Month YYYY&lt;/p&gt;</pre>@{param2}",
                     "docIDFormat": "Document identifier information <span class=\"rfc2119\">must</span> be presented in a <code>dl</code> list, where each <code>dt</code> element marks up an identifier role (\"This Version\", \"Latest Version\", \"History\", etc.) and each <code>dd</code> element includes a link whose link text is the identifier. That <code>dl</code> <span class=\"rfc2119\">must</span> itself be placed in a <code>details</code> element. <p><span style=\"font-style: italic\">Include this source code:</span><br/><code>&lt;details open&gt;&lt;summary&gt;More details about this document&lt;/summary&gt;&lt;dl&gt;...&lt;/dl&gt;&lt;/details&gt;</code></p>",
                     "docIDOrder": "Document identifier information <span class=\"rfc2119\">must</span> be present in this order: <ul><li>'This version' - URI to that version</li><li>'Latest version' - URI to the latest version. See also the (non-normative) <cite> <a href=\"https://www.w3.org/2005/05/tr-versions\">Version Management in W3C Technical Reports</a> </cite> for information about \"latest version\" URI and version management.</li><li class=\"historyuri\">'History' - URI to the history of the specification</li><li>Editor(s)</li><li>Feedback - GitHub repository issue links are required in the &lt;dl&gt;after &lt;dt&gt;Feedback:&lt;/dt&gt; in the headers (&lt;div class=\"head\"&gt;) of the document. Links are expected to be of the form <code>https://github.com/&lt;USER_OR_ORG&gt;/&lt;REPO_NAME&gt;/[issues|labels][/&hellip;]</code>.)</li></ul>",
-                    "docIDThisVersion": "The syntax of a “this version” URI <span class=\"rfc2119\">must</span> be <code>https://www.w3.org/TR/YYYY/@{param1}-shortname-YYYYMMDD/</code>.",
+                    "docIDThisVersion": "The syntax of a “this version” URI <span class=\"rfc2119\">must</span> be <code>https://www.w3.org/TR/YYYY/@{param1}-shortname-YYYYMMDD/</code>.@{param2}",
```

Comment:
To be on the safe side, I think the sentence regarding the shortname case should be there in all profiles so this also includes potential shortname changes